packages/node/src/index.ts

```diff
@@ -460,7 +460,26 @@ export const prepareCache: PrepareCache = ({ repoRootPath, workPath }) => {
 
 export const startDevServer: StartDevServer = async opts => {
   const { entrypoint, workPath, config, meta = {} } = opts;
-  const entryDir = join(workPath, dirname(entrypoint));
+  const entrypointPath = join(workPath, entrypoint);
+
+  if (config.middleware === true && typeof meta.requestUrl === 'string') {
```

Comment:
Curious: why do we need the `meta.requestUrl` here?

Reply:
To match against with the `matchers` 😄

Reply:
If you mean why are we checking if it's a string here, it's to appease TypeScript since `meta` keys are `uknown`.